diffyscan/utils/binary_verifier.py

```diff
@@ -0,0 +1,333 @@
+import hashlib
+import subprocess
+import json
+import os
+import stat
+import shutil
+
+from .common import fetch, pull, get_solc_native_platform_from_os
+from .helpers import create_dirs
+from .logger import logger
+from .encoder import encode_constructor_arguments
+from .constants import SOLC_DIR
+  
+def get_compiler_info(platform, required_compiler_version):
+    compilers_list_url = (
+      f'https://raw.githubusercontent.com/ethereum/solc-bin/gh-pages/{platform}/list.json'   
+    )
+    available_compilers_list = fetch(compilers_list_url).json()
+    required_build_info = next((compiler for compiler in available_compilers_list['builds'] if compiler['longVersion'] == required_compiler_version), None)
+
+    if not required_build_info:
+      raise ValueError(f'Required compiler version for "{platform}" is not found')
```

Comment:
For example?

Reply:
For this example, it may look something like this. This is a "nice to have" feature since `diffyscan,` I assume, can be used as a module and therefore should produce meaningful error types, not generic ones.  Disclaimer: I might be wrong, and this is not in the scope. cc @TheDZhon 

```python
class CompilerVersionNotFoundError(Exception):
    def __init__(self, platform, version):
        self.platform = platform
        self.version = version
        self.message = f'Required compiler version "{version}" for platform "{platform}" is not found'
        super().__init__(self.message)
        
...
raise CompilerVersionNotFoundError(platform, required_compiler_version)
```